File: 2019/s2.py
# code by savir singh
# 15/15 on DMOJ & CCC Grader

# check if prime
def checkprime(num):
    if num==1:
        return False
    if num==0:
        return False
    for z in range(2, int(num**0.5)+1):
        if (num%z) == 0:
            return False
    return True

# no. of test cases
t = int(input())

# loop through and begin the magic!
for i in range(t):
    n = int(input())
    a = n+1
    b = n-1
    while True:
        if checkprime(a) and checkprime(b):
            print(b, a)
            break
        a+=1
        b-=1
    

File: 2019/s2.cpp
// code by savir singh

#include <bits/stdc++.h>
#define ll long long
#define del erase
using namespace std;

bool prime(ll num)
{
    if (num==1 || num==0) {
        return false;
    }
    for (int i=2; i<pow(num, 0.5)+1; i++) {
        if (num%i==0) {
            return false;
        }
    }
    return true;
}

int main()
{
    int t;
    cin >> t;
    while (t--) {
        ll num;
        cin >> num;
        for (int i=0; i<num; i++) {
            if (prime(num-i) && prime(num+i)) {
                cout << num-i << " " << num+i << "\n";
                break;
            }
        }
    }
}
